RecoLocalTracker/SiPixelClusterizer/src/SiPixelFedCablingMapGPUWrapper.cc

```diff
@@ -41,21 +41,21 @@ SiPixelFedCablingMapGPUWrapper::SiPixelFedCablingMapGPUWrapper(SiPixelFedCabling
       for (unsigned int roc = 1; roc <= pixelgpudetails::MAX_ROC; roc++) {
         path = {fed, link, roc};
```

Comment:
is there a reason why at [line 39](https://github.com/cms-patatrack/cmssw/pull/301/files#diff-f8bcd30913e93b304e18b3037617ed84L39) we use
```c++
for (unsigned int fed = startFed; fed <= endFed; fed++) {
```
instead of
```c++
for (unsigned int fed: fedIds) {
```
?

Reply:
I don't know if it can happen, but if `fedIds` vector does not contain all the values between `fedIds.front()` and `fedIds.back()`, the result of those two is different.